ddtrace/bootstrap/sitecustomize.py

```diff
@@ -109,8 +109,9 @@ def update_patched_modules():
 
         patch_all(**EXTRA_PATCHED_MODULES)
 
-    if "DATADOG_ENV" in os.environ:
-        tracer.set_tags({constants.ENV_KEY: os.environ["DATADOG_ENV"]})
+    dd_env = get_env("env")
+    if dd_env:
```

Comment:
We should probably do a `is not None` instead here but I think all our other env var configuration reading uses the falsy check too so this is consistent.

Reply:
agreed that this isn't super necessary here. `DD_ENV=""` would be weird anyways. We can skip making any changes to this for now.